packages/vite/src/node/ssr/ssrExternal.ts

```diff
@@ -141,6 +143,14 @@ function collectExternals(
     // or are there others like SystemJS / AMD that we'd need to handle?
     // for now, we'll just leave this as is
     else if (/\.m?js$/.test(esmEntry)) {
+      const pkgPath = resolveFrom(`${id}/package.json`, root)
+      const pkgContent = fs.readFileSync(pkgPath, 'utf-8')
+
+      if (!pkgContent) {
+        continue
+      }
+      const pkg = JSON.parse(pkgContent)
```

Comment:
`pkg` and `pkgContent` shadow the variables of the same names in the outer scope. It might be error-prone.
Let's rename one of them to some other name to avoid confusion.

Reply:
done